packages/devtools_app/lib/src/ui/utils.dart

```diff
@@ -178,6 +182,27 @@ class _OffsetScrollbarState extends State<OffsetScrollbar> {
   }
 }
 
+void scrollToPosition(
```

Comment:
nit: rename to `maybeScrollToPosition` or `scrollIfNotVisible`. And then add dart doc explaining what this util method does.